src/main/java/core/basesyntax/FigureArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureArea {

```

Comment:
This is not a valid name for interface in Java. It should be an adjective or a meaningful noun - your figure is not a figure area.